ops/testing.py

```diff
@@ -1495,7 +1495,14 @@ def push(
 
     def list_files(self, path: str, *, pattern: str = None,
                    itself: bool = False) -> typing.List[pebble.FileInfo]:
-        files = [self._fs.get_path(path)]
+        try:
+            files = [self._fs.get_path(path)]
+        except FileNotFoundError as e:
+            # conform with the real pebble api
+            raise pebble.APIError(
+                body={}, message=str(e),
+                code=404, status='Not Found')
```

Comment:
Nit alert! This wrapping seems a little over-zealous, perhaps...

```suggestion
            raise pebble.APIError(
              body={}, message=str(e),code=404, status='Not Found')
```

I think the above would be more in-keeping with the style elsewhere in OF.

Reply:
good point!